docs/_documentations/remote-setupregistries.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: docs
+title: Adding a container registry in Codewind
+description: Adding a container registry in Codewind
+keywords: container, registry, IDE, guidance, image registry, Appsody, Docker, name, push registry, Kubernetes
+duration: 1 minute
+permalink: remote-setupregistries
+type: document
+---
+
+# Adding a container registry in Codewind
+
+## Prerequisite: Determining if you need a container registry 
+1\. First, determine if you need to add a container registry. The following scenarios in Codewind require you to specify which image registry is in use:
+- In the following scenario, configure Codewind with an image registry to push the application image to that specific image registry:
+    - Run Codewind on a remote Kubernetes cluster to develop a Codewind style project.
+- In the following scenario, configure Codewind to use the credentials for a specific image registry:
+    - Run Codewind on a remote Kubernetes cluster to develop an Appsody style project.
+    - The image registry for the Appsody stack requires credentials.
+- In the following scenario, from the command line, enter the `docker login` command in the registry before you create the Appsody project:
+    - Run Codewind locally to develop an Appsody style project.
+    - The image registry for the Appsody stack requires credentials. 
+If you do not develop any Codewind style projects, and you use an image registry that does not require credentials, you do not need to specify an image registry.
+2\. If you do need to add a container registry, start and run your IDE.
+
+## Adding a container registry in Codewind
+After your IDE is started and running, add the container registry to be used with Codewind.
+1. Select **Image Registry Manager**.
+  - **In Eclipse:** Right-click a connection in the Codewind Explorer view.
+  - **In VS Code:** From the Codewind Project Explorer view, right-click the remote connection or use the **Image Registry Manager** command on the command palette.
+2. Add your container registry.
+  - **In Eclipse:** From **Manage Image Registries**, click **Add...**.
+  - **In VS Code:** From the **Image Registry Manager**, click **Add New**.
+3. Enter the following information to add the registry:
+  - Registry server name, domain name, or address: `<registry-to-push-images-to>`
+  - User name: `<Your user name>`
+  - Password or API key: `<Your password or API key>`
+4. If you're using the newly added registry as a push registry, enter the namespace that you want to push your images to.
+  - For recommended values for common registries, see the [examples](#examples).
+5. Click **Designate as push registry** in Eclipse or **Select a Push Registry** in VS Code to determine which registry you use to push Codewind style projects to.
```

Comment:
In the Add dialog it is **Designate as push registry** but if the user is back in the Manage Image Registries dialog then it is **Set as Push**. Actually this is a bit confusing here. In the Add dialog they cannot enter a namespace unless they check **Designate as push registry** so it seems this should come before #4. If they have already added the registry and want to make it a push registry later then they can use the **Set as Push** button in the Manage Image Registries dialog.

Add dialog
- user must check **Designate as push registry** and then fill in a namespace

Manage Image Registries dialog
- the user can select a registry in the list and click **Set as Push**
![image](https://user-images.githubusercontent.com/9484320/75917230-af89ca80-5e27-11ea-8acb-4da5d3286882.png)